catalog/dags/maintenance/rotate_envfiles.py

```diff
@@ -281,14 +281,18 @@ def delete_stale_envfiles(
 
 
 @task
-def notify_complete(deleted_envfiles: dict[str, list[str]]):
+def notify_complete(
+    deleted_envfiles: dict[str, list[str]],
+    dag: DAG | None = None,
+):
     if deleted_envfiles:
```

Comment:
Nit/small improvement: Adding this would make the task more accurate regarding Airflow states. It requires an extra import:

```python
from airflow.exceptions import AirflowSkipException
```

```suggestion
    if not deleted_envfiles:
        raise AirflowSkipException("No environment files were deleted.")
```

Then, the rest of the function wouldn't need indentation. I couldn't add those changes here because GH says I cannot modify delete lines.

Reply:
Your suggestion makes total sense, implemented it in 91ff38c.